### PR TITLE
Implement API for saving bookmarks

### PR DESCRIPTION
**Pull request by rasendubi**
This addresses #1.

### CODE REVIEW

File: heutagogy/views.py [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
     return 'Hello, world!'
+
+@app.route('/api/bookmarks', methods=['POST'])
+def bookmark_post():
+    r = request.get_json()
+
+    try:
+        url = r['url']
+        title = r['title']
+        timestamp = r['timestamp']
```
Comment (Faradey27) [inline heutagogy/views.py:23]: should clients send timestamp or timestamp should be generated on server?

File: rest.http [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# This file contains example requests.
+# It can be used directly with Emacs restclient plugin.
+# See https://github.com/pashky/restclient.el
+#
+
+# Hello, world!
+GET http://127.0.0.1:5000
+
+# Save bookmark
+POST http://127.0.0.1:5000/api/bookmarks
```
Comment (Faradey27) [inline rest.http]: what about api version?
/api/v1 ...

File: heutagogy/views.py (hunk 1 above)
Comment (rasendubi) [reply heutagogy/views.py:23]: Clients (for now)

File: rest.http (hunk 2 above)
Comment (rasendubi) [reply rest.http]: Yep. I'll add it later

**Comment by rasendubi** [comment]
@Faradey27 addressed API versioning comment.

**Pull request merged**